src/components/Modal/Modal.jsx

```diff
@@ -118,31 +90,8 @@ class Modal extends React.Component {
         this.props.initialFocusSelector,
       );
       if (focusableElement) {
-        focusableElement.focus();
+        focusableElement.setAttribute('data-autofocus', 'true');
```

Comment:
This is a [react-focus-lock property](https://github.com/theKashey/react-focus-lock#autofocus) (used by react-focus-on) to set initial focus on a particular element.